test/module/irohad/torii/torii_service_test.cpp

```diff
@@ -383,9 +388,12 @@ TEST_F(ToriiServiceTest, CheckHash) {
     iroha::protocol::TxStatusRequest tx_request;
     tx_request.set_tx_hash(shared_model::crypto::toBinaryString(hash));
     iroha::protocol::ToriiResponse toriiResponse;
-    // when
-    client.Status(tx_request, toriiResponse);
-    // then
+    auto resub_counter(resubscribe_attempts);
+    do {
+      client.Status(tx_request, toriiResponse);
+    } while (toriiResponse.tx_hash()
+                 != shared_model::crypto::toBinaryString(hash)
```

Comment:
Please move `shared_model::crypto::toBinaryString(hash)` to separate variable to avoid duplication in line 398. Same with two test cases below.